src/main/java/com/epam/petclinic/clinic/model/Animal.java

```diff
@@ -0,0 +1,49 @@
+package com.epam.petclinic.clinic.model;
+
+import groovy.transform.EqualsAndHashCode;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+/**
+ * Animal entity.
+ * <p/>
+ * Date: 4/12/2017
+ *
+ * @author Stanislau Halauniou
+ */
+@Entity
+@Table(name="animal")
+@EqualsAndHashCode
```

Comment:
Replacement for equals and hashCode methods.  Should be discussed if we need it.